artemis-journal/pom.xml

```diff
@@ -15,57 +15,81 @@
+    </parent>
 
-   <artifactId>artemis-journal</artifactId>
-   <packaging>jar</packaging>
-   <name>ActiveMQ Artemis Journal</name>
+    <artifactId>artemis-journal</artifactId>
+    <packaging>jar</packaging>
+    <name>ActiveMQ Artemis Journal</name>
 
-   <properties>
-      <activemq.basedir>${project.basedir}/..</activemq.basedir>
-   </properties>
+    <properties>
+        <activemq.basedir>${project.basedir}/..</activemq.basedir>
+    </properties>
 
-   <dependencies>
-      <dependency>
-         <groupId>org.jboss.logging</groupId>
-         <artifactId>jboss-logging-processor</artifactId>
-         <scope>provided</scope>
-         <optional>true</optional>
-      </dependency>
+    <dependencies>
+        <dependency>
+            <groupId>org.jboss.logging</groupId>
+            <artifactId>jboss-logging-processor</artifactId>
+            <scope>provided</scope>
+            <optional>true</optional>
+        </dependency>
 
-      <!--
-          JBoss Logging
-      -->
-      <dependency>
-         <groupId>org.jboss.logging</groupId>
-         <artifactId>jboss-logging</artifactId>
-      </dependency>
-      <dependency>
-         <groupId>org.jboss.logmanager</groupId>
-         <artifactId>jboss-logmanager</artifactId>
-         <scope>test</scope>
-      </dependency>
-      <dependency>
-         <groupId>org.apache.activemq</groupId>
-         <artifactId>artemis-commons</artifactId>
-         <version>${project.version}</version>
-      </dependency>
-      <dependency>
-         <groupId>org.apache.activemq</groupId>
-         <artifactId>artemis-native</artifactId>
-         <version>${project.version}</version>
-      </dependency>
-      <!-- needed to compile the tests -->
-      <dependency>
-         <groupId>junit</groupId>
-         <artifactId>junit</artifactId>
-         <scope>test</scope>
-      </dependency>
-   </dependencies>
-</project>
+        <!--
+            JBoss Logging
+        -->
+        <dependency>
+            <groupId>org.jboss.logging</groupId>
+            <artifactId>jboss-logging</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.logmanager</groupId>
+            <artifactId>jboss-logmanager</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.activemq</groupId>
+            <artifactId>artemis-commons</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.activemq</groupId>
+            <artifactId>artemis-native</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <!-- needed to compile the tests -->
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>net.openhft</groupId>
```

Comment:
Hi Martyn,

i'm currently trying to fix the deps issue and i've found another problem:
my "tests" are not JUnit tests that could be run like the others, but harness benchmarks, thus in order to not invalidate the results they provides they defined a main method to be run eventually by the dev from the IDE.
They are dev tools to measure something and not something that (in this current form at least) could be embedded in a automatic build chain...
Hence there is no need to create different profiles that exclude these benchs using surefire's plugin, because surefire do not recognize them anyway as proper tests.
Considering these assumptions what do you recommend to do with them?
In this moment i've added them on the main pom with type of license and version (with scope test but only because "bench" doesn't exist in Maven!) and without version and licence type on pom of the journal module (with scope test).